test/functional/fixtures/api/es-next/wait/test.js

```diff
@@ -0,0 +1,40 @@
+var expect = require('chai').expect;
+
+
+describe('[API] Wait', function () {
+    describe('t.wait', function () {
+        it('Should wait the specified timeout', function () {
+            return runTests('./testcafe-fixtures/wait-test.js', 'Wait', { only: 'chrome' });
+        });
+
+        it('Should validate the timeout argument', function () {
+            return runTests('./testcafe-fixtures/wait-test.js', 'Incorrect timeout argument (wait)', { only: 'chrome' })
+                .catch(function (errs) {
+                    expect(errs[0]).to.contains('The timeout argument is expected to be a positive integer, but it was NaN.');
+                    expect(errs[0]).to.contains('> 17 |    await t.wait(NaN);');
+                });
+        });
+    });
+
+    describe('t.waitForElement', function () {
+        it('Should wait for the specified element', function () {
+            return runTests('./testcafe-fixtures/wait-test.js', 'Wait for element', { only: 'chrome' });
```

Comment:
But it's ok to keep error tests just for single browser, since their processing is implemented on server